### DISCUSSION TITLE
20240613 Weekly Development Meeting

### DISCUSSION THREAD

**Discussion by josecelano** [nautilus-cyberneering/.github#98] [Meetings]
- Time:  9:30 in Canary Islands (WET (UTC+0)).
- Time: 10:30 in Vaduz (CET (UTC+1)).

Location:
<https://meeting.blockfinance-eco.li/NautilusCyberneeringWeeklyDevMeeting>

Expected duration: 2 to 4 hours with breaks

Communicated attendants:

- Cameron Garnham(@da2ce7)  - 
- Jose Celano(@josecelano) - x
- Constantin Bosse(@cgbosse) - x
- Mario Melchor(@mario-nt) - x
- Graeme Byrne(@grmbyrn) - x

-------------------------------------------------------------

## Agenda:

1. Everyone will take their turn and:
- Go over his notes added as comments for the meeting which will have been added in advance to give an update of all the work done during the previous week indicating.
	- The issues worked on
	- The progress made on them
	- Other things done
- Inform of any faced challenges and request help if necessary to work on specific issues.
- Highlight the encountered difficulties, especially if there is a risk of them causing delays or affecting end product's quality.
- Communicate what will be the next week's tasks.
- List any new topics for further discussion in an additional Technical meeting that he noted for future discussion in comments to this discussion either before the meeting or after his turn so that they get considered.

During the meeting every team member will take their turns and follow this sequence.
The topics for further discussion will be noted in advance by each the team member and to the meetings agenda.

2. At the end of the meeting if there any new technical topics to be discussed further it will be decided to
set another meeting date for the discussion of these or to hold the meeting hereafter.

Also the persons to be part of the meeting will be decided on so that the others will be free to continue their own work.

**Comment by josecelano** [comment]
### Torrust Tracker

Issues:

- https://github.com/torrust/torrust-tracker/issues/884
- https://github.com/torrust/torrust-tracker/issues/770
- https://github.com/torrust/torrust-tracker/issues/659
- Update dependencies

### Torrust Index

Issues:

- https://github.com/torrust/torrust-index/issues/621
- https://github.com/torrust/torrust-index/issues/628
- https://github.com/torrust/torrust-index/issues/591 (**WIP**)
- Update dependencies

### Torrust Index GUI

Issues:

- https://github.com/torrust/torrust-index-gui/issues/570
- https://github.com/torrust/torrust-index-gui/issues/414

PR Review:

- https://github.com/torrust/torrust-index-gui/pull/573

### Torrust Org

- Review briefing for designer

**Comment by grmbyrn** [comment]
## Torrust Website
* [add megalinter and update dependencies #204](https://github.com/torrust/torrust-website/pull/204)

Issue:
* [Schema in deploy.yml #205](https://github.com/torrust/torrust-website/issues/205)

## Torrust Index GUI
* [add spinner gif to display instead of No results while data is loading #573](https://github.com/torrust/torrust-index-gui/pull/573)

## SvelteKit
* How to add Stripe
* How to add authentication
* Applied to [Frontend Mentor project](https://github.com/grmbyrn/art-gallery-website)

## Vue and Nuxt
* Vue [revision](https://www.youtube.com/watch?v=hNPwdOZ3qFU&t=201s&ab_channel=TraversyMedia)
* Learn about [Nuxt](https://www.youtube.com/playlist?list=PL8HkCX2C5h0XT3xWYn71TlsAAo0kizmVc)

**Comment by mario-nt** [comment]
### Index
	

- https://github.com/torrust/torrust-index/pull/619
- https://github.com/torrust/torrust-index/issues/615


### PR reviews

- https://github.com/torrust/torrust-index-gui/pull/568
- https://github.com/torrust/torrust-index/pull/619/files
- https://github.com/torrust/torrust-index-api-lib/pull/45
- https://github.com/torrust/torrust-index/issues/621

### Training/Research


- The Rust programming language book

### Other

- https://github.com/torrust/torrust-index/discussions/627
- One to one with @josecelano about error propagation in Rust
